agent/agent-api-public.yaml

```diff
@@ -5239,16 +5239,29 @@ definitions:
         minLength: 1
         maxLength: 80
         description: A unique identifier to ensure uniqueness of the datafeed.
-      filters:
+      eventTypes:
         type: array
         description: |
-          If type is set to be fanout, this should not be set.
-          At least one value is required if type datahose is specified. Values must be valid real event types, allowed values are:
-          * SOCIALMESSAGE
-          * CREATE_ROOM
-          * UPDATE_ROOM
+          At least one value is required if the type of feed is "datahose". Values must be a valid Real-Time Event type, i.e. one of:
```

Comment:
the same change should be applied to the agent-api-public-deprecated.yaml file as well. the deprecated one is a superset of this one.
Also we might want to change the version as well